src/content/development/contribution-guide/_index.en.md

```diff
@@ -78,11 +78,11 @@ To be expanded
 
 ### Unplanned Work
 
-To be expanded
-
 #### Ongoing Maintenance
 
-To be expanded
+Tasks that relate to ongoing maintenance (version bumps, image trimmings, CI and automation) will usually be small, unplanned tasks that
+typically occur throughout each release cycle. The associated GitHub issue / PR should be labeled as "maintenance".
```

Comment:
So a new label?

We already have `automation` label which fits at least the CI/automation stuff. Also version bumps in GHA can use this label.

Generally I think we need a label if it adds value otherwise we're just creating a plethora of them which dont add any real value.

Reply:
I think `maintenance` is meant to be a generalization of `automation`, which makes sense. I think I was the main person using that label, and I've switched to `maintenance` already.

Reply:
I am also using `automation` but I'm fine if we replace it with `maintenance`, as long as we don't have two which would be just confusing IMHO :)

Reply:
I'm replacing all `automation` labels with `maintenance` for open issues and will delete the `automation` label so theres no confusion